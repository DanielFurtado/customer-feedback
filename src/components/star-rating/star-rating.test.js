import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import StarRating from './star-rating';

describe(('Star Rating'), () => {
  it('renders', () => {
    const component = renderer.create(
      <StarRating rating={5} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
