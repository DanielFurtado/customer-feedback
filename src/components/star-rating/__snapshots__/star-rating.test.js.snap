// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Star Rating renders 1`] = `
<div
  className="sc-bcXHqe dMssKH"
  title="5 out of 5 stars"
>
  <div
    className="sc-gswNZR joqcDS Star--rating on"
  >
    <span
      className="star"
    >
      ★
    </span>
  </div>
  <div
    className="sc-gswNZR joqcDS Star--rating on"
  >
    <span
      className="star"
    >
      ★
    </span>
  </div>
  <div
    className="sc-gswNZR joqcDS Star--rating on"
  >
    <span
      className="star"
    >
      ★
    </span>
  </div>
  <div
    className="sc-gswNZR joqcDS Star--rating on"
  >
    <span
      className="star"
    >
      ★
    </span>
  </div>
  <div
    className="sc-gswNZR joqcDS Star--rating on"
  >
    <span
      className="star"
    >
      ★
    </span>
  </div>
</div>
`;
